DDD Workshop.ipynb

```diff
@@ -0,0 +1,547 @@
+    "\n",
+    "One of these that is particularly well-made is the [arcgis dashboard](https://www.arcgis.com/apps/opsdashboard/index.html#/bda7594740fd40299423467b48e9ecf6), which aggregates data and allows you to see the impact on your local county as well as the national/international trends. It does this without leaving room for p-hacking by allowing inappropriate filtering or manipulation, but still answering the questions it seeks to address. You can see the trendline for every state, as well as any country by selecting it in the list on the left. See here:\n",
+    "\n",
+    "![COVID](https://i.imgur.com/KGWmlY8.jpg)\n",
+    "\n",
+    "\n",
+    "<!-- \n",
+    "#### Census data\n",
+    "\n",
+    "The founding fathers of America were many things, but statisticians they were not. They did not have a particular fascination with regression, machine learning models, or gradient descent, and we shall shame them for that. However, they did appreciate the value of good data, err, data at least. From Article 1, section 2 of the U.S. Constitution:\n",
+    "\n",
+    "> Representatives and direct Taxes shall be apportioned among the several States which may be included within this Union, according to their respective Numbers, which shall be determined by adding to the whole Number of free Persons, including those bound to Service for a Term of Years, and excluding Indians not taxed, three fifths of all other Persons. The actual Enumeration shall be made within three Years after the first Meeting of the Congress of the United States, and within every subsequent Term of ten Years, in such Manner as they shall by Law direct.\n",
+    "\n",
+    "<!-- Cite: https://www.census.gov/history/pdf/Article_1_Section_2.pdf -->\n",
+    "\n",
+    "<!-- I don't want to make historical judgments on the moral quality of this data, but the statistical quality of the data is prevalent here. We see that there is 10-year periodic data-gathering, a full count of \"free Persons\", and a multiple on \"all other Persons.\" This means that the ratification of the 14th amendment (that rescinded the three-fifths compromise) in 1868 should be measurable in the differences in the 1860 and 1870 censuses.\n",
+    "\n",
+    "I DID NOT SEE THE CORRELATION I SUPPOSED HERE... I WILL INVESTIGATE FURTHER, BUT THIS SEEMS LIKE AN INTERESTING FOLLOW UP ARTICLE IF THE MYSTERY IS RESOLVED IN AN INTERESTING WAY-->\n",
+    "\n",
+    "What all of that text means is that every 10 years, falling on the years ending in 0, there is a nationwide tally of the people who live in the country. There is some historical context here of course as well, including the exception to Indian reservations that are not _really_ part of the U.S. in lots of ways, and the three-fifths compromise that was removed in the 14th amendment.\n",
```

Comment:
Looks like the other parts of the Census Data section were redacted, so i'm not sure if for this and the next section talking about the census you wanted to move the heading down here too?